### DISCUSSION TITLE
Is it possible to solve a variable stiffness problem?

### DISCUSSION THREAD

**Discussion by BraveDrXuTF** [kinnala/scikit-fem#1073] [Q&A] [answered]
Is it possible to solve a variable stiffness prolem, where the stiffness would change according to the spatial coordinates? For example, consider a Darcy type equation $\nabla k(\mathbf{x}) \nabla T=q$, where $\mathbf{x}$ is the spatial coordinates.

**Comment by gdmcbain** [comment, marked as answer]
Yes indeed; e.g.

https://github.com/kinnala/scikit-fem/blob/6159aa309cd182578405ee8a84553d9a6d7462de/docs/examples/ex16.py#L3

https://github.com/kinnala/scikit-fem/blob/6159aa309cd182578405ee8a84553d9a6d7462de/docs/examples/ex16.py#L32-L34

After integration by parts yours will look like this; _k_ (**x**) being the factor in parentheses.

**Reply by BraveDrXuTF** [reply]
Thank you!
